lib/peep.rb

```diff
@@ -0,0 +1,31 @@
+require 'pg'
+require_relative 'database_connection'
+
+class Peep
+  attr_reader :message, :date
+
+  def initialize(message:, date:)
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    result = DatabaseConnection.query("SELECT * FROM peeps;")
+    result.map do |peep|
+      Peep.new(message: peep['message'], date: peep['date'])
+    end
+  end
+
+  def self.create(message)
+    DatabaseConnection.query("INSERT INTO peeps (message, date)
+      VALUES($1, $2);", [message, Time.now.to_s[0, 9]])
```

Comment:
Clever way of getting the date! Another way to get the date would have been to use ALTER TABLE ADD COLUMN this_date DATE DEFAULT CURRENT_DATE; when creating the column on the table as it will get the date automatically once the a post is created.